core/src/org/samtools/htsjdk/core/HtsjdkException.java

```diff
@@ -0,0 +1,35 @@
+package org.samtools.htsjdk.core;
+
+/**
+ * Exception type for all exceptions caused at runtime by HTSJDK.

```

Comment:
Maybe more detailed.

```
Base exception type for all exceptions thrown by Htsjdk. 
All exceptions should extend from this.
```

If that's how we feel.  We should also decide about capitalization of htsjdk.